sh/publish.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
```

Comment:
I usually go with `set -euo pipefail`. The -u will cause bash to fail if you reference a variable that hasn't been defined. The -o pipefail will cause the entire pipe chain to fail if one of the parts of the pipe returns a non-zero exit code.